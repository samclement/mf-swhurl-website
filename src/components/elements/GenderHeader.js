import React from 'react'
import styled from 'styled-components'

const StyledH1 = styled.h1`
  width: 100%;
  text-align: center;
  text-transform: capitalize;
  margin-bottom: 0px;
  background-color: #eee;
`

export default props => {
  const { gender } = props
  return <StyledH1>{gender}</StyledH1>
}
