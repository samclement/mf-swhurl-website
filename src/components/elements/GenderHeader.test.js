/* eslint-env jest */

import React from 'react'
import { render } from '@testing-library/react'
import 'jest-styled-components'
import 'jest-dom/extend-expect'
import GenderHeader from './GenderHeader.js'

describe('With Snapshot Testing', () => {
  it('H1 shows "mf.swhurl.com"', () => {
    const { asFragment } = render(<GenderHeader gender="test" />)
    expect(asFragment()).toMatchSnapshot()
  })
})
