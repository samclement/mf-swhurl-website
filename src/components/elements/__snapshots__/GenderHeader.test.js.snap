// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`With Snapshot Testing H1 shows "mf.swhurl.com" 1`] = `
<DocumentFragment>
  .c0 {
  width: 100%;
  text-align: center;
  text-transform: capitalize;
  margin-bottom: 0px;
  background-color: #eee;
}

<h2
    class="c0"
  >
    test
  </h2>
</DocumentFragment>
`;
